<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cubipix</class>
 <widget class="QMainWindow" name="Cubipix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cubipix</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-image: url(:/files/images/background.png);
}

QPushButton {
	border: 1px solid rgb(85, 85, 85);
	border-radius: 10px;
	background-color: rgb(161, 210, 255);
}

QPushButton:hover {
	background-color: rgb(130, 174, 255);
}

QPushButton:pressed {
	background-color: rgb(85, 125, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="homeWidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>941</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Cubipix</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playLevelButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Jouer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="multiplayerButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Multiplayer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>300</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>360</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>A Propos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mapEditorButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Editeur de map</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="levelsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>941</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QPushButton" name="randomLevelButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Niveau al√©atoire</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="listButtonLevelsScrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>921</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>919</width>
        <height>429</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Niveaux</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backLevelsButton">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>941</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="mapView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>931</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="infoPlayer1Widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <widget class="QProgressBar" name="healthPlayer1ProgressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
     border: 1px solid grey;
     border-radius: 2px;
 }

 QProgressBar::chunk {
     background-color: rgb(185, 35, 63);
     width: 20px;
 }</string>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="namePlayer1Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Player1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="infoPlayer2Widget" native="true">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>20</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QProgressBar" name="healthPlayer2ProgressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
     border: 1px solid grey;
     border-radius: 2px;
 }

 QProgressBar::chunk {
     background-color: rgb(185, 35, 63);
     width: 20px;
 }</string>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="namePlayer2Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Player2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="playersListMultiplayerWidget" native="true">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>20</y>
       <width>171</width>
       <height>211</height>
      </rect>
     </property>
     <widget class="QListWidget" name="playersListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
	border: 0px;
	background-color: rgba(255, 255, 255, 0);
}</string>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <item>
       <property name="text">
        <string>Test</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Players</string>
      </property>
     </widget>
     <widget class="QLabel" name="numberPlayersLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>joueur(s)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="exitGameWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>931</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: rgba(0, 0, 0, 100);
}

QPushButton {
	border: 1px solid rgb(85, 85, 85);
	border-radius: 10px;
	background-color: rgb(161, 210, 255);
}

QPushButton:hover {
	background-color: rgb(130, 174, 255);
}

QPushButton:pressed {
	background-color: rgb(85, 125, 255);
}</string>
     </property>
     <widget class="QPushButton" name="backGameButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reprendre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsGameButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitGameButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>200</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="finishPartWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>931</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: rgba(0, 0, 0, 100);
}</string>
     </property>
     <widget class="QGroupBox" name="resultGameGroupBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	border: 1px solid gray;
	border-radius: 2px;
	margin-top: 1ex;
	background-color: rgba(0, 0, 0, 200);
 }
 
 QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding: 0 3px;
 }

QLabel {
	color: white;
	background-color: rgba(0, 0, 0, 0);
}

QProgressBar {
     border: 1px solid grey;
     border-radius: 2px;
 }

 QProgressBar::chunk {
     background-color: rgb(185, 35, 63);
     width: 20px;
 }

QPushButton {
	border: 1px solid rgb(85, 85, 85);
	border-radius: 10px;
	background-color: rgb(161, 210, 255);
}

QPushButton:hover {
	background-color: rgb(130, 174, 255);
}

QPushButton:pressed {
	background-color: rgb(85, 125, 255);
}</string>
      </property>
      <property name="title">
       <string>R√©sultats</string>
      </property>
      <widget class="QLabel" name="mentionResultLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Vous avez gagn√© !</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Joueur :</string>
       </property>
      </widget>
      <widget class="QLabel" name="usernamePlayerResultLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>120</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>username</string>
       </property>
      </widget>
      <widget class="QLabel" name="pixmapPlayerResultLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>71</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>pixmap</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="healthPlayerResultProgressBar">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>160</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
     border: 1px solid grey;
     border-radius: 2px;
 }

 QProgressBar::chunk {
     background-color: rgb(185, 35, 63);
     width: 20px;
 }</string>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Vie restante :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="continuerResultButton">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>310</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Continuer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="multiplayerModeWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>941</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>16</width>
       <height>281</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Local</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>En ligne</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pseudo :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mot de passe :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Les parties en local se jouent avec deux joueurs.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Joueur 1 :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="namePlayer1LineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Joueur 2 :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="namePlayer2LineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startLocalPartMultiplayerButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>D√©marrer la partie</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backMultiplayerButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Map :</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="mapMultiplayerSpinBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="multiplayerPartsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QTableView" name="partsMultiplayerTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>921</width>
       <height>431</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="statutRemoteServerLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Multiplayer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>D√©connexion</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameInfoLabel">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bienvenue, username !</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="logCubipix">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>20</y>
      <width>201</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
